src/dndbeyond/content-scripts/feat.js

```diff
@@ -0,0 +1,98 @@
+console.log("Beyond20: D&D Beyond Feats module loaded.");
+
+class FeatCharacter extends CharacterBase {
+    constructor(global_settings) {
+        super("feat", global_settings);
+    }
+    getDict() {
+        const dict = super.getDict();
+        if (this.avatar) {
+            dict.avatar = this.avatar;
+        }
+        return dict;
+    }
+}
+
+var character = null;
+var settings = getDefaultSettings();
+
+function addDisplayFeatButton() {
+    const icon32 = chrome.runtime.getURL("images/icons/badges/normal32.png");
+    const button = E.a({ class: "ct-beyond20-roll button-alt", href: "#" },
+        E.span({ class: "label" },
+            E.img({ class: "ct-beyond20-item-icon", src: icon32, style: "margin-right: 10px;" }),
```

Comment:
Since this is only to display the feat, and not do an actually dice roll, the button shouldn't include the beyond20 dice icon.

Reply:
Without the dice-roll icon the button looks very short and doesn't stick out as much (see image)

also the `items` pages have a dice as well so you should maybe consider modifying that page too since items don't always have an associated roll either

![image](https://user-images.githubusercontent.com/31316420/164738861-9b7cdd4a-eaa9-473e-b8d7-524f7d5fbb55.png)

Reply:
Agreed, that's 100% how I quickly find the Beyond20 settings when looking at my character sheet